main.js

```diff
@@ -0,0 +1,185 @@
+let button = document.getElementById('postSubmit');
+
+// Post submit button event listener
+button.addEventListener('click', function () {
+  let usersName = document.getElementById('users-name').value;
+  let usersPost = document.getElementById('users-post').value;
+
+  // Ensures both fields are filled out
+  if (!usersName || !usersPost) {
+    alert("Please fill out both the name and the post fields.");
+    return;
+  }
+
+  // Wrapper div for grouping all post-related elements
+  let postWrapper = document.createElement('div');
+  postWrapper.classList.add('post-wrapper');
+  postWrapper.style.marginBottom = '20px'; // Optional spacing between posts
```

Comment:
Instead of hardcoding styles, you could use Bootstrap’s spacing utilities (mt-2, mb-3)